octobot_trading/enums.py

```diff
@@ -59,6 +59,20 @@ class OrderStatus(Enum):
     FILLED = "filled"
     CANCELED = "canceled"
     CLOSED = "closed"
+    UNKNOWN = "unknown"
+
+
+class OrderStates(Enum):
```

Comment:
Should we add an `REFRESHING` state or something like this to signify that an order is being updated (during open/closed orders updater update for example) ? Or maybe it can be an internal attribute to any state ? The idea is to prevent operations on an order that is being udpated by an updater that is waiting for the exchange answer (this update not being an order operation like cancel/close but just a regular order update to check it's alright on the exchange side)

Reply:
Good idea. Something between a pending state and the completed state :+1: